controllers/storagecluster/quickstart_constants.go

```diff
@@ -84,8 +85,13 @@ spec:
 
   2. Click Create Object Bucket Claim.
 
-  3. Specify a name for your claim and select an appropriate storage class for your application: - to use on-premises object storage, select ocs-storagecluster-ceph-rgw. - to use Multicloud Object Gateway, select openshift-storage.noobaa.io.
+  3. Specify a name for your claim and select an appropriate storage class for your application:  
+
+        **to use on-premises object storage,** select ocs-storagecluster-ceph-rgw.  
+
+        **to use Multicloud Object Gateway,** select openshift-storage.noobaa.io.  
 
+  
```

Comment:
Why 2 new lines? Wouldn't 1 suffice?

Reply:
same reason as above. it is for spacing out the lines otherwise we get an output like this
![Screenshot from 2021-01-12 19-33-31](https://user-images.githubusercontent.com/57935785/104324245-352cdb00-550d-11eb-89ad-810cc16e8752.png)